app/controllers/api/v1/users_controller.rb

```diff
@@ -1,7 +1,9 @@
 module Api
   module V1
     class UsersController < ApplicationController
-      def index; end
+      def index
+        render json: User.last(10)
```

Comment:
Do we have any json serializer for this association?

Reply:
You mean a model?